src/pages/components/inline.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Inline
+description: >
+  The Inline loader is used as a typing indicator. This signals the other party
+  is present and preparing a reply.
+package: '@zendeskgarden/react-loaders'
+components:
+  - loaders/src/elements/Inline.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>As a typing indicator</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        <Markdown>
+          To show a loading state, use the
+          [Dots](https://zendeskgarden.github.io/react-components/loaders/#dots-1) loader instead
+        </Markdown>
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+The Inline loader is used to represent someone typing.
+
+import Default from '../../examples/loaders/inline/InlineDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/loaders/inline/InlineDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Color
+
+By default the color will be inherited from the parent element’s color. Can be
```

Comment:
Nice curly quote. Also a reminder that the `gatsby-remark-smartypants` plugin will pick up any that get missed 😉